crates/uv/tests/pip_install_scenarios.rs

```diff
@@ -501,7 +501,7 @@ fn dependency_excludes_range_of_compatible_versions() {
 /// There is a non-contiguous range of compatible versions for the requested package
 /// `a`, but another dependency `c` excludes the range. This is the same as
 /// `dependency-excludes-range-of-compatible-versions` but some of the versions of
-/// `a` are incompatible for another reason e.g. dependency on non-existent package
+/// `a` are incompatible for another reason e.g. dependency on non-existant package
```

Comment:
These typos are in the scenarios... fix over in packse instead of editing these generated files please.

Reply:
ref https://github.com/astral-sh/uv/pull/2804